impeller/renderer/backend/vulkan/swapchain/ahb/ahb_texture_pool_vk.h

```diff
@@ -45,7 +45,7 @@ class AHBTexturePoolVK {
   explicit AHBTexturePoolVK(
       std::weak_ptr<Context> context,
       android::HardwareBufferDescriptor desc,
-      size_t max_entries = 2u,
+      size_t max_entries = 3u,
```

Comment:
This count is the number of textures idling and used neither by the client (us) or the compositor. I don't think holding onto an additional texture matters here.

Reply:
Removed. I think I must be seeing the case where the texture ages quicker than expected.